clients/ops/admin-ui/src/features/datastore-connections/add-connection/AddConnection.tsx

```diff
@@ -18,35 +18,11 @@ import { AddConnectionStep } from "./types";
 const AddConnection: React.FC = () => {
   const dispatch = useDispatch();
   const router = useRouter();
-  const { connectorType, key, step: currentStep } = router.query;
+  const { connectorType, step: currentStep } = router.query;
 
   const { connectionOption, step } = useAppSelector(selectConnectionTypeState);
 
-  /**
-   * NOTE: If the user reloads the web page via F5, the react redux store state is lost.
```

Comment:
The following code snippet was no longer needed given the Redux store is stored in localStorage.